### PR TITLE
Bg api testing 159323179

### PR DESCRIPTION
**Pull request by supershegs**
### What does this PR do?
Making correction base on feedback

 **Description of task that has been implemented.**
- Used ES6 array iterator in my EntryModel.js file.
- Add content to my README.md file.
- Created data to test PUT /entries/entries_id

 **How should this be manually tested?**
- To test using jasmine  run npm test on your terminal.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -11,7 +11,9 @@
         "import/no-unresolved": "off",
         "no-var": "off",
         "vars-on-top":"off",
-        "prefer-destructuring": "off"
+        "class-methods-use-this":"off",
+        "prefer-destructuring": "off",
```
Comment (fidelisojeah) [inline .eslintrc]: Remove this,

File: server/controller/Entries.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import EntryModel from '../model/EntriesModel';
 const Entries = {
     create(req, res) {
         const newEntry = EntryModel.add(req.body);
+        console.log(req.body);
```
Comment (fidelisojeah) [inline server/controller/Entries.js]: This is here cos?

File: server/controller/Entries.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import EntryModel from '../model/EntriesModel';
 const Entries = {
     create(req, res) {
```
Comment (fidelisojeah) [inline server/controller/Entries.js]: req, res?
Please use better descriptions for your variables `request`, `response`

File: server/model/EntriesModel.js [hunk 4]
```diff
@@ -1,47 +1,53 @@
...
     }
 
     findOne(id) {
-        for (let i = 0; i < this.entries.length; i += 1) {
-            if (this.entries[i].id === id) {
-                return this.entries[i];
-            }
-        }
-        return {};
+        for (const entry of entries) {
```
Comment (fidelisojeah) [inline server/model/EntriesModel.js]: entries is an array, use es6 array iterators an example: `entries.forEach()` ??

**Review by fidelisojeah — CHANGES_REQUESTED** [review]

File: server/model/EntriesModel.js (hunk 4 above)
Comment (supershegs) [reply server/model/EntriesModel.js]: ok, thanks will work on it

File: server/controller/Entries.js (hunk 3 above)
Comment (supershegs) [reply server/controller/Entries.js]: ok, thanks am working on it

File: .eslintrc (hunk 1 above)
Comment (supershegs) [reply .eslintrc]: ok i will remove

File: server/controller/Entries.js (hunk 2 above)
Comment (supershegs) [reply server/controller/Entries.js]: I used it for test, will remove it

**Review by fidelisojeah — APPROVED** [review]
You are good to go

**Pull request merged**